releases/release-1.20/README.md

```diff
@@ -71,6 +72,7 @@ The 1.20 release cycle is proposed as follows:
 | **KubeCon CloudNativeCon North America Starts** | Community | Tue, November 17 |  | |
 | **KubeCon CloudNativeCon North America Ends** | Community | Friday, November 20 |  | |
 | Docs deadline - PRs ready for review | Docs Lead | Mon November 23 | week 11 | |
+| **[Test Freeze]** (EOD PST) | Branch Manager | Mon November 23 | |
```

Comment:
Down a couple lines "Code Thaw" ought to become "Code and Test Thaw"?  Or just "Thaw" and link to release_phases.md#thaw with that getting yet another clarification from it's current "Code Thaw" being generic thawing for frozen stuff like code and test.

Reply:
OK, I can update the phases document. I think a more generic `Thaw` makes sense:

Reply:
This text looks reasonable to me.

Reply:
@tpepper take a look at the updated phases document please.

Reply:
Actually, need to fix the references in handbooks.